themes/MUG/invenio.cfg

```diff
@@ -277,17 +294,60 @@ OVERRIDE_RIGHT_SECTION_CONTACT_EMAIL = "rdmsupport@medunigraz.at"
 OVERRIDE_RESOURCE_OVERVIEW = True
 """Resource overview section"""
 
-THEME_FOOTER_TEMPLATE = "invenio_override/footer_mug.html"
-"""footer template"""
+# ============================================================================
+# Invenio-override - Global Search Configuration
+# ============================================================================
+# Combine default components with global search components
+OVERRIDE_RDM_RECORDS_SERVICE_COMPONENTS = RDMDefaultRecordsComponents + [
+    RDMToGlobalSearchComponent
+]
+OVERRIDE_LOM_RECORDS_SERVICE_COMPONENTS = LOMDefaultRecordsComponents + [
+    LOMToGlobalSearchComponent
+]
+OVERRIDE_MARC21_RECORDS_SERVICE_COMPONENTS = Marc21DefaultRecordsComponents + [
+    Marc21ToGlobalSearchComponent
+]
```

Comment:
do you need these also in instance?

Reply:
You’re right, I’ll remove the redundant lines from the instance configuration...